tests/chromium.e2e.runner.js

```diff
@@ -20,3 +21,4 @@ async function loadChromiumDriver () {
 const driver_chrome = await loadChromiumDriver();
 
 legacyRayCharles.runTests(driver_chrome);
+nonLegacyZim.runTests(driver_chrome);
```

Comment:
Just a quick comment on why this doesn't work. You can't use the same driver instance, because it will be quit before the second test suite is run. You need to make a new instance, e.g. `const driver_chrome_test2 = await loadChromiumDriver();`.